components/core/buttons/TextButton.vue

```diff
@@ -53,14 +58,35 @@ export default {
                 '--secondary': this.secondary,
                 '--rounded': this.rounded,
                 '--block': this.block,
+                '--is-link': this.isLink,
             }
         },
     },
+    watch: {
+        isLink: {
+            immediate: true,
+            handler() {
+                if (this.href) {
+                    this.isLink = true
+                } else if (this.to) {
+                    this.isLink = true
+                } else {
+                    this.isLink = false
+                }
+            },
+        },
+    },
```

Comment:
1. We normally combine the usage of `data` (Line 48 ~ 52) and `watch` into a `computed` instance.
2. The `if-else` logic of the `isLink` could be simplified as `this.href || this.to`

Therefore, it's as simple as:

```suggestion
    computed: {
        isLink() {
            return this.href || this.to
        },
    },
```

Reply:
Okay, I got it.